cypress/e2e/files.spec.ts

```diff
@@ -2,7 +2,7 @@ import "cypress-file-upload"
 import { Interceptors, TEST_REPO_NAME } from "../fixtures/constants"
 
 describe("Files", () => {
-  const FILE_TITLE = "singapore"
+  const FILE_TITLE = "singapore-file"
```

Comment:
seems liek the same file might be reused across `editPage` and `files`, maybe we should consider having both of the specs refer to the same constant by exporting it from our fixtures file? 

this means that a change to filename in `editPage` might cause a downstream failure still

Reply:
We actually want the files referred to be different - the tests were made independently of each other, and the issue is that running them sequentially mistakenly referred to the file created in `editPage` which had a similar title, but was not meant to be the same file

Reply:
oh ok i see!